examples/arduino/LVGL_Arduino/LVGL_Arduino.ino

```diff
@@ -21,6 +21,7 @@
 
 /*LVGL draw into this buffer, 1/10 screen size usually works well. The size is in bytes*/
 #define DRAW_BUF_SIZE (TFT_HOR_RES * TFT_VER_RES / 10 * (LV_COLOR_DEPTH / 8))
+uint8_t draw_buf[DRAW_BUF_SIZE];
```

Comment:
```suggestion
static uint32_t draw_buf[DRAW_BUF_SIZE / 4];
```

And update below code to 
```c
disp = lv_tft_espi_create(TFT_HOR_RES, TFT_VER_RES, draw_buf, sizeof(draw_buf));
```

Reply:
updated. `uint32_t` might be easier to read when debugging :D

Reply:
Yes. And it's also required by the software renderer.